src/common/python.ts

```diff
@@ -17,87 +67,66 @@ interface IExtensionApi {
             execCommand: string[] | undefined;
         };
     };
+    environment: {
+        getActiveEnvironmentPath(resource?: Uri | undefined): Promise<EnvPathType | undefined>;
+        getEnvironmentDetails(
+            path: string,
+            options?: EnvironmentDetailsOptions,
+        ): Promise<EnvironmentDetails | undefined>;
```

Comment:
As this API will go away soon, I think it's best only to rely on `getActiveEnvironmentPath` for now.

Reply:
```suggestion
```